docs/settings.rst

```diff
@@ -52,6 +52,21 @@ to DynamoDB.
 
 .. _Session: http://docs.python-requests.org/en/master/api/#request-sessions
 
+allow_rate_limited_scan_without_consumed_capacity
+-------------------------------------------------
+
+Default: ``False``
+
+If ``True``, ``rate_limited_scan()`` will proceed silently (without
+rate limiting) if the DynamoDB server does not return consumed
+capacity information. This can be useful when running tests against an
+implementation that does not support returning consumed capacity
+information, such as `DynamoDB Local
+<http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.html>`_.
+
+You may wish to consider use of `dynalite
```

Comment:
Can we include this recommendation (and a note about this setting) in `local.rst` instead?